ShopParser/ComputerParsers/AvitelaComputerParser.cs

```diff
@@ -70,14 +70,15 @@ public List<Computer> ParseShop()
 
 
         //parses laptop window, updates computer fields
-        public Computer ParseWindow(string url)
+        public async Task<Computer> ParseWindow(string url)
```

Comment:
No await operator

Reply:
it is redundant for now. but i am planning to optimize parsing even more, so several one shop Windows will be parsed at the same time. there i will use await keyword. if my optimization will not work, i will delete async from here